x/concentrated-liquidity/msg_server_test.go

```diff
@@ -159,7 +160,7 @@ func (s *KeeperTestSuite) TestAddToPosition_Events() {
 	}{
 		"happy path": {
 			expectedAddedToPositionEvent: 1,
-			expectedMessageEvents:        5,
+			expectedMessageEvents:        4,
```

Comment:
Do we know what caused the drop here?

Reply:
Now that we do an early return on spread rewards claim when there are no spread rewards, we emit one less event.